module.txt

```diff
@@ -1,9 +1,9 @@
 {
     "id" : "MarcinScIncubator",
     "version" : "0.1.1-SNAPSHOT",
-    "author" : "",
+    "author" : "MarcinSc, Cervator, Josharias",
     "displayName" : "MarcinScIncubator",
-    "description" : "",
+    "description" : "A module for adding a generic incubator in Terasology",
```

Comment:
This module does NOT add a "generic incubator" - it's an "incubator" for marcin's ideas. (What it actually contains is an HTML parser/renderer engine.)